crates/vision/src/ball_detection.rs

```diff
@@ -332,18 +358,40 @@ fn project_balls_to_ground(
     clusters: &[BallCluster],
     camera_matrix: &CameraMatrix,
     ball_radius: f32,
-) -> Vec<Ball> {
+    measurement_noise: Vector2<Pixel>,
```

Comment:
Rename: -> `resting_measurement_noise`

Reply:
No! The measurement noise should be independent of the ball state. I think the naming is fine and the type explains which coordinate system the noise is given in

Reply:
Makes sense 👌🏻